sucury.py

```diff
@@ -173,27 +173,43 @@ def update(self):
             self.head.y += self.ymov * GRID_SIZE
 
 ##
-## The apple class.
+## The fruit class.
 ##
 
-class Apple:
+class Fruit:
     def __init__(self):
 
         # Pick a random position within the game arena
         self.x = int(random.randint(0, WIDTH)/GRID_SIZE) * GRID_SIZE
         self.y = int(random.randint(0, HEIGHT)/GRID_SIZE) * GRID_SIZE
 
-        # Create an apple at that location
+        # Create an fruit at that location
         self.rect = pygame.Rect(self.x, self.y, GRID_SIZE, GRID_SIZE)
 
+        self.type = random.randint(0, 3)
+
     # This function is called each interation of the game loop
 
-    def update(self):
+    def update(self, CLOCK_TICKS=6):
 
-        # Drop the apple
-        pygame.draw.rect(arena, APPLE_COLOR, self.rect)
+        # Drop the fruit
+        # Apple maintains snake's speed
+        if self.type == 0:
+            pygame.draw.rect(arena, APPLE_COLOR, self.rect)
 
+        # Pear increases snake's speed
+        elif self.type == 1:
+            pygame.draw.rect(arena, PEAR_COLOR, self.rect)
 
+        # Blueberry decreases snake's speed
+        elif self.type == 2:
+            pygame.draw.rect(arena, BLUEBERRY_COLOR, self.rect)
+
+        # Orange makes snake grow 2 segments
+        elif self.type == 3:
+            pygame.draw.rect(arena, ORANGE_COLOR, self.rect)
+
```

Comment:
in the current implementation, the fruit types are indicated by numbers (0, 1, 2, 3) in the if-elif statements. it would enhance code readability and maintainability to replace these numbers with named constants. for instance, define constants like APPLE = 0, PEAR = 1, BLUEBERRY = 2, and ORANGE = 3 at the beginning of your code. then, replace the numbers in these conditions with the respective constants. this change would make the code more understandable and easier to maintain, as it clearly indicates what each number represents without needing to refer back to the comments or documentation